packages/app/src/cli/commands/app/logs.ts

```diff
@@ -0,0 +1,91 @@
+import {appFlags} from '../../flags.js'
+import Command from '../../utilities/app-command.js'
+import {showApiKeyDeprecationWarning} from '../../prompts/deprecation-warnings.js'
+import {checkFolderIsValidApp} from '../../models/app/loader.js'
+import {logs} from '../../services/logs.js'
+import {Flags} from '@oclif/core'
+import {globalFlags} from '@shopify/cli-kit/node/cli'
+import {addPublicMetadata} from '@shopify/cli-kit/node/metadata'
+
+export default class Logs extends Command {
+  static summary = 'Stream detailed logs for your Shopify app.'
+
+  static descriptionWithMarkdown = `
+  Opens a real-time stream of detailed log events from the selected app and store. Use the \`--source\` argument to limit output to a particular log source, such as a Shopify Function or webhook topic. Use the \`sources\` subcommand to list available sources.
+  The \`--json\` argument can be used to receive log entries as line-delimited JSON (JSONL). By piping the output to tools like \`jq\`, you can filter the output to specific information.
+  \`\`\`
+  shopify app logs --json
+  \`\`\`
+  `
+
+  static description = this.descriptionWithoutMarkdown()
+
+  static flags = {
+    ...globalFlags,
+    ...appFlags,
+    'api-key': Flags.string({
+      hidden: true,
+      description: 'The API key of your app.',
+      env: 'SHOPIFY_FLAG_APP_API_KEY',
+      exclusive: ['config'],
+    }),
+    json: Flags.boolean({
+      hidden: false,
+      description: 'Print logs in JSON format.',
+      env: 'SHOPIFY_FLAG_JSON',
+      default: false,
+    }),
```

Comment:
This flag appears to be unsupported- should we make a follow-up ticket for this and remove it from here for now?

Reply:
Ah didn't know that, that is good with me.